src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,70 @@
+.stopwatch {
+  width: 80vmin;
+  height: 80vmin;
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: $borderWatch;
+  border-radius: 50%;
+
+  &__center {
+    width: 5vmin;
+    height: 5vmin;
+    background-color: $colorCenter;
```

Comment:
The variable `$colorCenter` should be written in kebab-case as `$color-center` to follow the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules).